selftest/version/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "C"
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+
+	"github.com/aquasecurity/libbpfgo"
+)
+
+func main() {
+
+	cmd := exec.Command("git", "describe", "--tags")
+	cmd.Dir = "../../libbpf"
+
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// libbpf doesn't put the patch version in exported version
+	// symbols, so use just prefix to exclude it
+	if strings.HasPrefix(libbpfgo.VersionString(), string(b)) {
```

Comment:
When I first read libbpfgo.VersionString() I thought we were comparing libbpf version with libbpfgo version (to see if libbpf string was contained in libbpfgo string). Should we change "VersionString" name ?

Reply:
Sure!